droidlet/tools/hitl/dashboard_app/dashboard_frontend/src/component/pipeline/metaInfo/runList.js

```diff
@@ -126,24 +136,81 @@ const RunList = (props) => {
         socket.on("get_job_list", (data) => handleReceivedRunList(data));
     }, [socket, handleReceivedRunList]);
 
+    const onSearch = (value) => {
+        setRangeValue([]); // reset filter value
+        if (value) {
+            setDisplayData(runListData.filter((o) =>
+            (o.name.includes(value)
+                || o.batch_id === value
+                || o.description.includes(value)
+            )));
+        } else {
+            setDisplayData(runListData);
+        }
+    }
+
+    const filterOnTime = (value) => {
```

Comment:
`value` is a bit confusing here, use a more explicit name or document this param